lib/html/includes.js

```diff
@@ -592,6 +592,31 @@ var MiniProfiler = (function () {
           };
         }
 
+        // https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+        if(typeof(window.fetch) === 'function' && window.fetch.__patchedByMiniProfiler === undefined) {
+          var __originalFetch = window.fetch;
+
+          console.warn('alert: patching window.fetch for profiling');
```

Comment:
not sure about the logging though. plus some may not even have console.warn

maybe we just have an option for patching fetch if you select that then fetch is patched?

Reply:
Sure, I will exclude the warning.

Is there already a mechanism for conditionally excluding certain `included` integrations? 

My thinking was, if you are using fetch and you are using rack-mini-profiler, you want the integration. If you aren't using fetch and/or the browser does not support fetch—i.e. `window.fetch` is undefined, there is no issue.

What do you think @SamSaffron ?